docs/glossary.md

```diff
@@ -0,0 +1,184 @@
+---
+id: glossary
+title: Glossary
+sidebar_label: Glossary
+description:  Avail common reference for general definitions and concepts.
+
+keywords:
+    - docs
+    - avail
+    - modularity
+    - Monolithic
+    - data availability
+image: https://availproject.github.io/img/avail/AvailDocs.png
+
+
+
+---
+
+# Avail Glossary
+## General
+
+### Modular Blockchain
+A modular blockchain is one that focuses on handling a select few duties and outsources the rest to one or more separate layers.
+
+### Monolithic 
+In a monolithic chain, all tasks are handled on a single layer or a group of tightly coupled chains operating on the same layer. The latter distinction is important to note: a network of interconnected blockchains that handle all roles, such as Polkadot’s parachains or Avalanche’s subnets, doesn’t qualify as modular architecture.
+
+### Data Availability
+Data availability is the guarantee that block proposers publish all transaction data of a block, along with making this data available to all network participants. In essence, it refers to all nodes’ ability to access and download all the data contained within the various blocks of a network.
+
+### Scalability
+Blockchain scalability refers to a blockchain's capacity to execute transactions, store data, and come to consensus as the network's user base grows.
+
+### Execution
+Execution is how nodes on the blockchain process transactions to transition the blockchain between states. Nodes participating in consensus must execute transactions using their copy of the blockchain to attest before validating blocks.
```

Comment:
Again, this is a stock blockchain answer that does not describe Avail very well.

Avail, as a modular base chain, does not have a general purpose execution layer. Execution happens in other layers (e.g. rollups) and the data is posted to Avail as raw data only, not to be executed.

"Consensus" means something more limited in Avail than in blockchains with an enshrined execution layer. In Avail, it means the chain agrees that data is being properly published. That means quite explicitly that validator nodes _do not_ execute transactions before attesting that blocks are valid (except for a small handful like balance transfers), they are _only_ attesting that blocks are correctly packaging up the published data. This is one key reason why Avail can grow block size — Validators are doing less work for each block, so growing the block size does not impact them as much as in other blockchains.